Sources/Contour/Filters/IMessageExchangeFilterDecorator.cs

```diff
@@ -1,7 +1,7 @@
-﻿namespace Contour.Filters
-{
-    using System.Threading.Tasks;
+﻿using System.Threading.Tasks;
 
+namespace Contour.Filters
+{
     public interface IMessageExchangeFilterDecorator
```

Comment:
A type which should be used as a decorator must implement an interface of the decorated object per definition. See http://www.oodesign.com/decorator-pattern.html for detailed explanation. The purpose of this is to hide the actual difference between the original object and the augmented one from the consumers of the interface. Why does IMessageExchangeFilterDecorator not do it?